openlibrary/templates/covers/author_photo.html

```diff
@@ -6,7 +6,7 @@
 $if author_thumbnail_url:
     <div class="author coverMagic">
         <div class="SRPCover bookCover">
-            <a href="#seeImage" class="coverLook" title="Pull up a larger author photo"><img src="$author_thumbnail_url" class="cover" alt="Photo of $author.name"/></a>
+            <a href="#seeImage" class="coverLook dialog--open" title="Pull up a larger author photo"><img src="$author_thumbnail_url" class="cover" alt="Photo of $author.name"/></a>
```

Comment:
Can we use the aria?

```suggestion
            <a aria-controls="seeImage" class="coverLook dialog--open" title="Pull up a larger author photo"><img src="$author_thumbnail_url" class="cover" alt="Photo of $author.name"/></a>
```

Reply:
Done